packages/redux-interop/src/lib/redux-interop.ts

```diff
@@ -0,0 +1,140 @@
+import type { Unit, Store, EventCallable } from 'effector';
+import type { Store as ReduxStore, Action } from 'redux';
+import {
+  createStore,
+  createEvent,
+  is,
+  sample,
+  attach,
+  scopeBind,
+} from 'effector';
+
+/**
+ *
+ * Utility function to create an Effector API to interact with Redux store,
+ * useful for cases like soft migration from Redux to Effector.
+ *
+ * @param config {reduxStore: ReduxStore, setup: Unit<any>} - reduxStore is a redux store, setup is an effector unit which will setup subscription to the store
```

Comment:
```suggestion
 * @param config - interop config
 * @param config.reduxStore - a redux store
 * @param config.setup - effector unit which will setup subscription to the store
```

https://jsdoc.app/tags-param